src/runtime/crypto/utils.ts

```diff
@@ -0,0 +1,65 @@
+import {crypto} from './crypto';
+
+export async function createSHA256HMAC(
+  secret: string,
+  payload: string,
+): Promise<string> {
+  const enc = new TextEncoder();
+  const key = await crypto.subtle.importKey(
```

Comment:
This doesn't seem to exist before node 18. Is there an alternative to `crypto.subtle` for previous versions that we can use?